packages/@o3r/core/schematics/index.it.spec.ts

```diff
@@ -154,7 +154,7 @@ describe('new Otter application', () => {
     addImportToAppModule('TestAddConfigComponentContModule', 'src/components/test-add-config-component');
 
     execSync(`yarn ng g @o3r/core:component --defaults=true test-add-theming-component ${defaultOptions}`, execAppOptions);
-    execSync('yarn ng g @o3r/styling:add-theming --defaults=true --path="src/components/test-add-theming-component/presenter/test-add-theming-component-pres.style.scss"', execAppOptions);
+    execSync('yarn ng g @o3r/styling:add-theming --defaults=true --path="src/components/test-add-theming-component/presenter/test-add-theming-component-pres.component.scss"', execAppOptions);
```

Comment:
It's the ng generator which is adding this suffix ? Should we do a `move` to rename the file and keep it with ``.style``, the same you've done for `spec`